src/task2.java

```diff
@@ -0,0 +1,98 @@
+public class task2 {
+    public task2() {
+    }
+
+    static void stringToArrayToArrayToString() {
+        String stringToReverse = "Test String";
```

Comment:
each method uses a string "Test String" instead of the string provided in the task "I'm going to go to school today morning." 
you should use the same input string across all methods to maintain consistency and fulfill the task's requirements